js/index.js

```diff
@@ -69,7 +134,7 @@ document.addEventListener("DOMContentLoaded", () => {
       const entry = e.target.parentNode;
       messageList.removeChild(entry);
       if (messageList.length === true) {
```

Comment:
Does this work for you? I would never expect `length` to be `true`... but if it's non-zero, it's true-ish, and you could do this:
```suggestion
      if (messageList.length) {
```

Reply:
... this might also be related to the message removal bug.

Reply:
thanks for the note. Once I made the suggested change it wasn't hiding my header after removing all the messages. I realized I needed to call (messageList.children.length) in the if statement. Now it's working properly. 🙌